tests/commands/utils_tests.py

```diff
@@ -57,7 +57,8 @@ def setUpModule():
 
     with gpg.core.Context(armor=True) as ctx:
         # Add the public and private keys. They have no password
-        search_dir = os.path.join(os.path.dirname(__file__), '../static/gpg-keys')
+        search_dir = os.path.join(
```

Comment:
linebreaks like these are the reason why I would not blindly push everything that autopep8 proposes.